install/Docker/dockerfiles/scripts/setup_kafka.sh

```diff
@@ -55,7 +55,7 @@ configure_kafka_properties()
 	update_param "zookeeper.connect" ${KAFKA_ZOOKEEPER_CONNECT}
 	update_param "listeners" ${KAFKA_LISTENERS}
 	update_param "advertised.listeners" ${KAFKA_ADV_LISTENERS}
-       	update_param "listener.security.protocol.map" "SSL:SSL"
+       	update_param "listener.security.protocol.map" "SSL:SSL,EXTERNAL:SSL"
```

Comment:
How will this behave in Docker only setup ??

Reply:
We are just extending the configuration which holds good with Docker as well as the non-docker deployment